examples/Makefile.am

```diff
@@ -1,27 +1,29 @@
-noinst_PROGRAMS = meshlinkapp chat chatpp manynodes channels monitor groupchat
+noinst_PROGRAMS = meshlinkapp chat manynodes channels groupchat
 
 AM_CPPFLAGS = $(PTHREAD_CFLAGS) -I${top_srcdir}/src -iquote. -Wall
 AM_LDFLAGS = $(PTHREAD_LIBS)
 
 meshlinkapp_SOURCES = meshlinkapp.c
-meshlinkapp_LDADD = ${top_builddir}/src/libmeshlink.la
+meshlinkapp_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
 chat_SOURCES = chat.c
-chat_LDADD = ${top_builddir}/src/libmeshlink.la
+chat_LDADD = ${top_builddir}/src/libmeshlink.la ${top_builddir}/catta/src/libcatta.la
 
-chatpp_SOURCES = chatpp.cc
-chatpp_LDADD = ${top_builddir}/src/libmeshlink.la
+#noinst_PROGRAMS += chatpp
+#chatpp_SOURCES = chatpp.cc
+#chatpp_LDADD = ${top_builddir}/src/libmeshlink.la
```

Comment:
I managed to compile chatpp for Android, but you have to tell `./configure` to use the right C++ compiler, like so:
```
CC=/tmp/my-android-toolchain/bin/arm-linux-androideabi-gcc CXX=/tmp/my-android-toolchain/bin/arm-linux-androideabi-g++ ./configure --host=arm-linux-androideabi
```

Reply:
Also, `chatpp_LDADD` needs Catta added.

Reply:
I actually did that before building with `CXX=/tmp/my-android-toolchain/bin/arm-linux-androideabi-g++`, but even though in my system it gave fatal error as it didn't find `new` header file which I think some module is missing?

Reply:
Hm, I didn't get any errors about the `new` header file, but I did get an error because `meshlink_set_connection_try_cb` was not in `src/meshlink.sym`. If I add that, `chatpp` compiles.